src/components/Error/Error.component.tsx

```
@@ -0,0 +1,35 @@
+import { useNavigate } from 'react-router-dom';
+
+import { Box, Stack, Typography, useTheme } from '@mui/material';
+
+import { Button } from '@components/Button';
+import { APP_ROUTES } from '@constant';
+
+import { ErrorPropsType } from './Error.types';
+
+export const Error = ({ img, title, description }: ErrorPropsType) => {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)